README.md

```diff
@@ -53,6 +53,12 @@ Reach project contributors via these channels:
  * [Gitter chat room](https://gitter.im/jaegertracing/Lobby)
  * [Github issues](https://github.com/uber/jaeger/issues)
 
+## Adopters
+
+Jaeger is [used in production](./ADOPTERS.md) by organizations such as Atlassian, Orange, Red Hat and Uber.
```

Comment:
I think it's a fair point. There are different types of users on this list
  * some have full prod deployment
  * some are using Jaeger end to end, but in dev/qa (which could also a valid use case)
  * some are using parts of Jaeger like the client libs
  * some are using Jaeger as part of other products (e.g. vendors who may not even want to reveal their actual customers)
  * some are just experimenting

Rather than listing them as "Adopters", maybe we can name the readme section "Testimonials" and point to the survey (#207)?